scripts/kind-e2e/e2e.sh

```diff
@@ -238,7 +238,7 @@ function create_subm_vars() {
   ce_ipsec_ikeport=500
   ce_ipsec_nattport=4500
 
-  subm_ns=operators
+  subm_ns=submariner
```

Comment:
We could alternatively change the default Operator namespace to "operators", but we'd also need to update docs. In general, I just think the default for the Operator should be what we use for E2E.